doc/source/whatsnew/v1.1.0.rst

```diff
@@ -245,6 +245,65 @@ If needed you can adjust the bins with the argument ``offset`` (a Timedelta) tha
 
 For a full example, see: :ref:`timeseries.adjust-the-start-of-the-bins`.
 
+.. _whatsnew_110.floating:
+
+Experimental nullable data types for float data
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We've added :class:`Float32Dtype` / :class:`Float64Dtype` and :class:`~arrays.FloatingArray`,
+an extension data type dedicated to floating point data that can hold the
+``pd.NA`` missing value indicator (:issue:`32265`, :issue:`34307`).
+
+While the default float data type already supports missing values using ``np.nan``,
+this new data type uses ``pd.NA`` (and its corresponding behaviour) as missing
+value indicator, in line with the already existing nullable :ref:`integer <integer_na>`
+and :ref:`boolean <boolean>` data types.
+
+One example where the behaviour of ``np.nan`` and ``pd.NA`` is different is
+comparison operations:
+
+.. code-block:: python
+
+  # the default numpy float64 dtype
+  >>> s1 = pd.Series([1.5, None])
+  >>> s1
+  0    1.5
+  1    NaN
+  dtype: float64
+
+  >>> s1 > 1
+  0     True
+  1    False
+  dtype: bool
+
+  # the new nullable float64 dtype
+  >>> s2 = pd.Series([1.5, None], dtype="Float64")
+  >>> s2
+  0    1.5
+  1   <NA>
+  dtype: Float64
+
+  >>> s2 > 1
+  0    True
+  1    <NA>
+  dtype: boolean
+
+See the :ref:`missing_data.NA` doc section for more details on the behaviour
+when using the ``pd.NA`` missing value indicator.
+
+As shown above, the dtype can be specified using the "Float64" or "Float32"
+string (capitalized to distinguish it from the default "float64" data type).
+Alternatively, you can also use the dtype object:
+
+.. ipython:: python
+
+   pd.Series([1.5, None], dtype=pd.Float32Dtype())
+
+.. warning::
+
+   Experimental: the new floating data types are currently experimental, and its
```

Comment:
is there a doc section can link to?

Reply:
IMO, we should consolidate https://pandas.pydata.org/pandas-docs/dev/user_guide/integer_na.html and https://pandas.pydata.org/pandas-docs/dev/user_guide/boolean.html into a single "nullable data types" page and add it there (not in this PR).

Reply:
Indeed, that has been on my personal to do list, but never got to it. In general we should have a page about data types (and can still have some subpages if needed)